common/src/main/java/com/wynntils/models/character/SkillPointModel.java

```diff
@@ -352,20 +356,16 @@ private boolean checkTomesUnlocked(ContainerContent content) {
         return LoreUtils.getStringLore(content.items().get(TOME_SLOT)).contains("✔");
     }
 
-    private boolean verifyCharacterInfo(
-            ContainerContent content, Int2ObjectMap<ItemStack> changes, ContainerContentChangeType changeType) {
+    private boolean verifyChange(
+            ContainerContent content,
+            Int2ObjectMap<ItemStack> changes,
+            ContainerContentChangeType changeType,
+            int soulPointItemSlot) {
         // soul points resent last for both containers
```

Comment:
I don't understand this comment. Is it a typo?

Reply:
No, it means the soul point item (nether star or diamond axe) will be sent as the last item in the stream of item updates, when some container is opened

Reply:
Aha, re-sent, not resent. I get it.